core/message-tags-3.3.md

```diff
@@ -105,7 +105,9 @@ Servers MAY apply moderation to this command using existing or newly specified m
 
 Servers MUST NOT deliver `TAGMSG` to clients that haven't negotiated the message tags capability.
 
-Servers SHOULD reject any `TAGMSG` command sent without tags. In this case, they MUST use the `ERR_NEEDMOREPARAMS` (`461`) error numeric.
+Servers SHOULD reject any `TAGMSG` command sent without tags, or where there are no tags with the client-only prefix
+and all tags without the client-only prefix are removed by the server.
```

Comment:
Hmm, I think we can simplify this down a bit to something like:

> Servers SHOULD reject any `TAGMSG` command sent without tags, or where all tags on the message end up being removed by the server.

Even if it's not as explicit, I think it's clearer than the current text (mainly because I need to go back and re-read the current sentence a few times before I understand what's actually meant by it, even though I already know).